src/main/webapp/features.jsp

```diff
@@ -1,184 +1,238 @@
-<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
 <t:staticPage currentPage="features">
     <div class="container">
         <div class="lead">
-            Here is an overview of some notable TEAMMATES features.
-            The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.
+	        <h1 class="color_orange">
+	            Features
+	        </h1>
+            <p style="font-size:14px"> Here is an overview of some notable TEAMMATES features.
+            The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.</p>
         </div>
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Team peer evaluations</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    Have a team project in your course? Set up a 'team peer evaluation session' for students to give anonymous peer feedback to team members.
-                    <br><br>
-                    Students can provide confidential peer evaluations to you too.
+                <img class="img-responsive col-xs-12" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations" style="height: 75%; width: 75%;">
+                <div class="row">
+                	<div class="row">
+		                <div class="col-xs-12 col-sm-7 col-sm-offset-1">
```

Comment:
Just a quick comment: is there a better way to achieve this effect instead of wrapping the element in two `row` classes? If no other css classes currently present in the app helps with this, then it might be possible to create a new css class (say `features-page-text` as an example name) with custom padding.